internal/cluster/workflow/delete_k8s_resources.go

```diff
@@ -0,0 +1,83 @@
+// Copyright © 2019 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package workflow
+
+import (
+	"time"
+
+	"github.com/goph/emperror"
+	"go.uber.org/cadence/workflow"
+)
+
+const DeleteK8sResourcesWorkflowName = "delete-k8s-resources"
+
+type DeleteK8sResourcesWorkflowInput struct {
+	K8sConfig []byte
```

Comment:
`Force bool` option should be added (later?)

Reply:
This workflow doesn't need to know whether the operation is being forced or not, because it only affects how the failure of this workflow is handled by the caller.

Reply:
I added a logic to ignore some of the helm step's legitimate errors. This solves my main concern.